src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonPropertyInfoOfT.cs

```diff
@@ -43,78 +43,89 @@ internal sealed class JsonPropertyInfo<T> : JsonPropertyInfo
             bool isVirtual,
             JsonConverter converter,
             JsonIgnoreCondition? ignoreCondition,
-            JsonNumberHandling? parentTypeNumberHandling,
```

Comment:
Am I right to understand that this parameter was not being used anywhere?

Reply:
yes, we assign it in the `Configure()` as it could get out of sync once we get into contract resolving which will allow to change this (also I think it might have not been used before my previous PR as well)